### PR TITLE
Bring Oberon PSA core v1.2.1.1

### PR DESCRIPTION
**Pull request by Vge0rge**
Brings the new Oberon PSA core version 1.2.1.1 which includes the new standardized PAKE APIs.

Ref: NCSDK-26057

test-sdk-nrf: pake_api_alignment

### CODE REVIEW

**Review by mswarowsky — APPROVED** [review]

**Review by tejlmand — APPROVED** [review]

**Review by vili-nordic — CHANGES_REQUESTED** [review]
Can you also remove PSA_CRYPTO_SPAKE2P_USE_VERSION_04 in Kconfig

**Comment by Vge0rge** [comment]
> Can you also remove PSA_CRYPTO_SPAKE2P_USE_VERSION_04 in Kconfig

You are right of course, I removed it.

File: samples/crypto/ecjpake/src/main.c [hunk 1]
```diff
@@ -122,30 +122,36 @@ psa_status_t pake_setup(psa_pake_operation_t *op, psa_pake_cipher_suite_t *cs, c
...
-	}
-
 	return PSA_SUCCESS;
 }
 
 psa_status_t do_key_derivation(psa_pake_operation_t *op, uint8_t *key_buffer,
 			       size_t key_buffer_size)
 {
+	psa_status_t status = PSA_ERROR_CORRUPTION_DETECTED;
+	mbedtls_svc_key_id_t key;
```
Comment (vili-nordic) [inline samples/crypto/ecjpake/src/main.c:132]: This should really be `psa_key_id_t`. I think you had some issues with that? If that truly does not work. can you add a comment about why this is not according to psa spec? (psa_key_id_t is used other places in this file)

**Review by vili-nordic — APPROVED** [review]

File: samples/crypto/ecjpake/src/main.c (hunk 1 above)
Comment (mswarowsky) [reply samples/crypto/ecjpake/src/main.c]: Yeah got some warning during the change but was due to my setup, changed it now to `psa_key_id_t`

**Comment by jfischer-no** [comment]
@Vge0rge You need to rebase, please also get some +1 in https://github.com/nrfconnect/sdk-mbedtls/pull/30.

**Review by frkv — APPROVED** [review]
LGTM

**Review by mia-ko — APPROVED** [review]

**Review by ArekBalysNordic — APPROVED** [review]

**Comment by jfischer-no** [comment]
@Vge0rge please rebase (sdk-zephyr version updated in https://github.com/nrfconnect/sdk-nrf/pull/14235)

File: west.yml [hunk 2]
```diff
@@ -61,7 +61,7 @@ manifest:
     # https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/zephyr/guides/modules.html
     - name: zephyr
       repo-path: sdk-zephyr
-      revision: f3f276dd2ecad981a2abf1d87c3d3d0b2e9d5373
+      revision: 76b0463a5a4b8b44cb4c82e5906711d18ae439db
```
Comment (jfischer-no) [inline west.yml:64]: 0a1854bb2c1eba17490d1c82ba1f749a5745fc1e
https://github.com/nrfconnect/sdk-zephyr/commit/0a1854bb2c1eba17490d1c82ba1f749a5745fc1e

Comment (Vge0rge) [reply west.yml]: Updated, please merge.

Comment (Vge0rge) [reply west.yml]: @jfischer-no  If you want me to change the hash yet again you just edit the comment I don't any notification unfortunately. Please ping me if you want me to a a change. 
I will update to: 7735bc6d171ab2fe6590ab47171f5decdbe2ddfd

**Comment by stephen-nordic** [comment]
The crypto test suite failure is unrelated to the changes here (crypto had passed in a previous run)

**Comment by Vge0rge** [comment]
We had other CI instabilities with IOT samples which I rerun and now they are green. Since I did that I also rerun Crypto and it is also green right now.

**Pull request merged**